src/StockDetails/index.jsx

```diff
@@ -34,6 +35,7 @@ const StockDetails = () => {
         visible={A2WPopupVisible}
         onClose={() => setA2WPopupVisible(false)}
       />
+      <TransactionPopup />
```

Comment:
this popup should be integrated just like to watchlist popup at the top. The stock details component should handle its visibility state